src/models/components/agent/llm.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Model representing a large language model (LLM) configuration used by the agent.
+ */
+export class LLMModel {
+  constructor(data = {}) {
+    /**
+     * Type of the LLM (e.g., GPT, Claude).
+     * @type {string}
+     */
+    this.type = data.type || '';
+
+    /**
+     * Name or identifier of the LLM model.
+     * @type {string}
+     */
+    this.model = data.model || '';
+
+    /**
+     * @type {number}
+     */
+    this.temperature = data.temperature || 0.7;
+
+    /**
+     * @type {number}
+     */
+    this.seed = data.seed || 1;
```

Comment:
The default values of 0.7 and 1 are misleading. The actual default value of the temperature in the backend is 0. Can we leave these as `data.temperature || ''` and `data.seed || ''` ('' is probably an empty string, the temperature is float and the seed is int, so I'm not sure how to specify the empty values) as the other properties?